### ISSUE TITLE
Cut line to smart clip conversion (likely code cleanup)

### ISSUE THREAD

**Issue by LWinterberg** [audacity/audacity#7717]
### Description

Cut lines have mostly been technically obsoleted since 3.1, but do cause issues whenever they crop up. With the UX improvements to ripple editing (#2893) they no longer are necessary.

Starting with 4.0, all Cut lines should be converted into individual clip boundaries. Nested cut lines can be ignored, content hidden by nested cut lines becomes normal hidden smart clip data. 

When a project is opened which contains cut lines, the following message is shown: 

> ### Cut lines have been converted to smart clips
>
> Audacity versions since 4.0.0 no longer support cut lines. Content hidden by cut lines has been converted to hidden smart clip content. Please verify that everything works as intended before saving.
>
> [OK]

**Comment by Tantacrul** [comment]
I don't believe this is something we even have anymore in Au4? Perhaps it's now a code cleanup task only?
